setup.cfg

```diff
@@ -12,10 +12,21 @@ addopts =
     --ignore examples
     --ignore docs
     --doctest-modules
+    --durations 10
+    --timeout 600
+    --cov sktime
+    --cov-report xml
+    --cov-report html
+    --showlocals
+    --pyargs
```

Comment:
I don't think this is needed since the invocation does not pass any arguments

Reply:
ok, I didnt know what `--pyargs` is used for, but let me then remove it